docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -131,10 +128,11 @@ quarkus:
           port: 8082
 ----
 
-If you enable the `staging` profile, the HTTP port will be 8082, whereas it would be 8081 otherwise.
+If you enable the `staging` profile, the HTTP port will be `8082`, whereas it would be `8081` otherwise.
```

Comment:
```suggestion
If you enable the `staging` profile, the HTTP port is set to `8082` instead of `8081`.
```
or .. if the reader needs to action this:
```suggestion
If you enable the `staging` profile, you must set the HTTP port to `8082`, instead of `8081`.
```